### PR TITLE
Minor change: 	Show link to quickly copy-paste when adding file

### PR DESCRIPTION
**Pull request by Foxcool**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e298c9c9c7018872b9edec5be1025c6fa6b413ca
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

 * Commit: 368d62f807ad9d83257766c79f00e6728ace45da
  * Invalid signoff. Commit message must end with  
    **License: MIT  
Signed-off-by: .* <.*>**

We ask for a few features in the commit message for Open Source licensing hygiene and commit message clarity.
`git commit --amend` can often help you quickly improve the commit message.
[Guidelines](https://github.com/ipfs/community/blob/master/contribution-guidelines.md#commit-messages) and [a script](https://github.com/ipfs/community/blob/master/dev/tools/hooks/setup_commit_msg_hook.sh) are available to help in the long run.
Your feedback on GitCop is welcome on [this issue](https://github.com/ipfs/infrastructure/issues/23).

---
This message was auto-generated by https://gitcop.com

**Comment by jbenet** [comment]
@Foxcool you can `git push -f origin <branchname>` here once you fix. no need for a different PR

File: core/commands/add.go [hunk 1]
```diff
@@ -266,7 +266,7 @@ remains to be implemented.
 					if quiet {
 						fmt.Fprintf(res.Stdout(), "%s\n", output.Hash)
 					} else {
-						fmt.Fprintf(res.Stdout(), "added %s %s\n", output.Hash, output.Name)
+						fmt.Fprintf(res.Stdout(), "added %s %s\nhttps://ipfs.io/ipfs/%s\n", output.Hash, output.Name, output.Hash)
```
Comment (jbenet) [inline core/commands/add.go]: sorry, this is an incompatible change. Build a tool on top of `ipfs add` to do this please. see how i do it in: https://github.com/jbenet/ipfs-screencap/blob/master/ipfs-screencap#L88-L94

**Comment by jbenet** [comment]
@Foxcool nevermind, see comment at https://github.com/ipfs/go-ipfs/pull/2218/files#r50087160 -- thanks but this belongs on top.

**Comment by Foxcool** [comment]
Thanks. I will try to deal with it better. (:

**Comment by jbenet** [comment]
@Foxcool thank you for the contribution though :) :+1:

**Pull request closed without merging**